### DISCUSSION TITLE
Improved Aggregator Support

### DISCUSSION THREAD

**Discussion by khmarbaise** [support-and-care/maven-support-and-care#13] [Ideas for Funding]
Currently a number of plugins or setups require an aggregator setup (for example JaCoCo, JavaDoc, PITest) but at the moment it hard to handle that correctly. There are some solutions but they are not satisfactory. In JaCoCo typically the resulting report will be *aggregated* into a final module (EAR, SpringBoot main application etc.). Also the handling of the dependencies is not very good.  Or you have to create a supplemental `reporting` module ... (which results in manual maintaince).

* Other parts which require a better aggregator support?

* More thinking about it TBD...

**Comment by ascheman** [comment]
This discussion has been converted to Epic(s) for tracking:
- #46
- #47

Closing this discussion.
